packages/airnode-node/src/evm/handlers/fetch-pending-requests.ts

```diff
@@ -45,10 +45,9 @@ export async function fetchPendingRequests(state: ProviderState<EVMProviderState
     withdrawals: withdrawalRequests,
   };
 
-  // Block any requests that cannot be processed
-  // 1. API calls related to a wallet with a pending withdrawal cannot be processed
-  const [blockLogs, updatedRequests] = blocking.blockRequestsWithWithdrawals(groupedRequests);
-  logger.logPending(blockLogs, baseLogOptions);
+  // Block (filter out) any requests that cannot be processed
+  const [blockRequestsLogs, allowedRequests] = blocking.blockRequests(groupedRequests);
```

Comment:
The `allowedRequests` suggest that all of these requests are "pending", but this is not necessarily true, because some requests may be flagged as Ignored in `verifyApiCallIds` (line 37)

Reply:
I will actually need to change this since I ended up "marking" the requests instead of filtering them.

I will think about the naming while doing this.